cloudmock/aws/mockautoscaling/group.go

```diff
@@ -76,10 +153,44 @@ func (m *MockAutoscaling) TerminateInstanceInAutoScalingGroup(input *autoscaling
 		for i := range group.Instances {
 			if aws.StringValue(group.Instances[i].InstanceId) == aws.StringValue(input.InstanceId) {
 				group.Instances = append(group.Instances[:i], group.Instances[i+1:]...)
-				return nil, nil
+				return &autoscaling.TerminateInstanceInAutoScalingGroupOutput{
+					Activity: nil, // TODO
+				}, nil
 			}
 		}
 	}
 
 	return nil, fmt.Errorf("Instance not found")
 }
+
+func (m *MockAutoscaling) DescribeAutoScalingGroupsWithContext(aws.Context, *autoscaling.DescribeAutoScalingGroupsInput, ...request.Option) (*autoscaling.DescribeAutoScalingGroupsOutput, error) {
+	glog.Fatalf("Not implemented")
```

Comment:
You always say to not use duplicate log messages. When not implemented. I think flatal returns a err. Might as well just return an err here.

Reply:
glog.Fatalf does a stack-trace, so we're OK :-)